services/core-web/src/components/dashboard/varianceHomePage/VarianceHomePage.js

```diff
@@ -305,33 +305,31 @@ export class VarianceHomePage extends Component {
           <h1>Browse Variances</h1>
         </div>
         <div className="landing-page__content">
-          <div className="page__content">
```

Comment:
Removing this div makes this component look consistent with the other home page components.